src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsAmazonObject.java

```diff
@@ -17,6 +18,7 @@
 public class AdsAmazonObject extends AdsBaseObject {
 
   private static final String AMAZON_SCRIPT_URL = "amazon-adsystem.com/e/dtb";
+  private static final String AMAZON_BID_URL = "aax.amazon-adsystem.com/e/dtb/bid";
```

Comment:
Could we re-use `AMAZON_SCRIPT_URL` or change `AMAZON_SCRIPT_URL` to `aax.amazon-adsystem.com/e/dtb/bid`?